### PR TITLE
Add template for the translation

### PR DESCRIPTION
**Pull request by BatoolMM**
### Summary

This is a template for the translation made in [Crowdin](https://turingway.crowdin.com/turing-way), related to #2365. It's based largely on the already existed templates with minor changes.

### List of changes proposed in this PR (pull-request)

* *Adding new issue template for the translation so others can request translation through GH*

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+assignees: ''
+
+---
+
+<!--
+Please complete the following sections when you open an issue. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. If you have write access to the repository please also assign the appropriate label (or labels) to your issue. Note that text within html comment tags will not be rendered.
+-->
+### Summary
+
+<!-- Please provide a detailed description of the the langauge that you want to translate to and the chapters that you would like to translate.
```
Comment (acocac) [inline .github/ISSUE_TEMPLATE/Request-Translation:15]:
```suggestion
<!-- Please provide a detailed description of the language that you want to translate to and the chapters that you would like to translate. 
```
Fix some typos

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+-->
+
+*Lorem ipsum dolor sit amet, consectetur adipiscing.*
+
+### What is the target language?
+
+<!-- Is your langauge added to Crowdin or you need to be added?
+We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->
+
+- [ ] *Target langauge is .*
```
Comment (acocac) [inline .github/ISSUE_TEMPLATE/Request-Translation:27]:
```suggestion
- [ ] *Target language is .*
```

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- [ ] *I wish to be manager in Crowdin to entries for this language.*
```
Comment (acocac) [inline .github/ISSUE_TEMPLATE/Request-Translation:28]:
```suggestion
- [ ] *I wish to be manager in Crowdin for entries in this language.*
```
No sure what you mean by entries

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- [ ] *I will only proofread translation for this language.*
+
+
+### What is the sections/chapter you want to translate?
+
+<!-- We have priority list for the translation but if you wish to translate other chpaters outside the priority list, please indicate here
```
Comment (acocac) [inline .github/ISSUE_TEMPLATE/Request-Translation:34]:
```suggestion
<!-- We have a priority list of chapters for translation, but if you wish to translate other(s), please indicate here
```
Fix typo and shorter sentence

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->
+
+- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
+- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
+
+### Who can help?
+
+<!-- We suggest using bullets (indicated by * or -) and filled checkboxes [x] here.
+Will this be an individual effort or you have other individuals/Community who will oversee the translation.
```
Comment (acocac) [inline .github/ISSUE_TEMPLATE/Request-Translation:43]:
```suggestion
Will this be an individual effort or you have other individuals/Community who will oversee the translation?
```
add question mark

**Review by acocac — DISMISSED** [review]
@BatoolMM great work. It would be very useful to start new translations in TTW. I just made some minor suggestions.

**Comment by BatoolMM** [comment]
> @BatoolMM great work. It would be very useful to start new translations in TTW. I just made some minor suggestions.

Thank you @acocac for your review!

**Review by malvikasharan — APPROVED** [review]

**Comment by malvikasharan** [comment]
@all-contributors please add @BatoolMM for maintenance and doc

**Comment by malvikasharan** [comment]
@all-contributors please add @acocac  for maintenance and doc

**Review by timothy22000 — COMMENTED** [review]
Hi @BatoolMM,

Apologies for the delay. I left some comments when your PR was first opened but forgot to click "Submit Review" so all my comments didn't show!

Best Regards,
Tim

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+assignees: ''
+
+---
+
+<!--
+Please complete the following sections when you open an issue. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. If you have write access to the repository please also assign the appropriate label (or labels) to your issue. Note that text within html comment tags will not be rendered.
+-->
+### Summary
+
+<!-- Please provide a detailed description of the language that you want to translate to and the chapters that you would like to translate.
```
Comment (timothy22000) [inline .github/ISSUE_TEMPLATE/Request-Translation:15]: "language" -> "language/languages"?

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+### Summary
+
+<!-- Please provide a detailed description of the language that you want to translate to and the chapters that you would like to translate. 
+Please provide as much context as possible and link to related issues and/or pull requests.
+Providing links to sections you're referencing for clarity is highly encouraged.
+-->
+
+*Lorem ipsum dolor sit amet, consectetur adipiscing.*
+
+### What is the target language?
```
Comment (timothy22000) [inline .github/ISSUE_TEMPLATE/Request-Translation:22]: "Target langauge is ." -> "Target language/languages is/are"? - just a thought

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+<!-- Is your langauge added to Crowdin or you need to be added?
```
Comment (timothy22000) [inline .github/ISSUE_TEMPLATE/Request-Translation:24]: minor typo. "langauge" -> "language"

File: .github/ISSUE_TEMPLATE/Request-Translation (hunk 6 above)
Comment (BatoolMM) [inline .github/ISSUE_TEMPLATE/Request-Translation:15]:
```suggestion
<!-- Please provide a detailed description of the language/languages that you want to translate to and the chapters that you would like to translate. 
```

File: .github/ISSUE_TEMPLATE/Request-Translation (hunk 7 above)
Comment (BatoolMM) [inline .github/ISSUE_TEMPLATE/Request-Translation:22]:
```suggestion
### What is the target language/languages?
```

File: .github/ISSUE_TEMPLATE/Request-Translation (hunk 8 above)
Comment (BatoolMM) [inline .github/ISSUE_TEMPLATE/Request-Translation:24]:
```suggestion
<!-- Is your language added to Crowdin or you need to be added?
```

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->
+
+- [ ] *Target language is .*
```
Comment (BatoolMM) [inline .github/ISSUE_TEMPLATE/Request-Translation:27]:
```suggestion
- [ ] * Target language/languages is/are .*
```

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+- [ ] *I wish to be manager in Crowdin for entries in this language.*
+- [ ] *I will only proofread translation for this language.*
+
```
Comment (BatoolMM) [inline .github/ISSUE_TEMPLATE/Request-Translation:30]: - [ ] Do you have a local translation to be uploaded to the Translation Memory?

File: .github/ISSUE_TEMPLATE/Request-Translation [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+
+### What is the sections/chapter you want to translate?
```
Comment (BatoolMM) [inline .github/ISSUE_TEMPLATE/Request-Translation:32]:
```suggestion
### What is the sections/chapters you want to translate?
```

[the remark this answers is not in the document]
Comment (BatoolMM) [reply .github/ISSUE_TEMPLATE/Request-Translation]: > Hi @BatoolMM,
> 
> Apologies for the delay. I left some comments when your PR was first opened but forgot to click "Submit Review" so all my comments didn't show!
> 
> Only realized it today when I was wondering why there was no response. >.<
> 

Don't worry at all, thank you so much, Timothy for taking the time to review it. Your input added a great value to the template and the translation. Let me know if I missed any of the changes you requested. 

Thank you again!

**Comment by malvikasharan** [comment]
Thank you all. This looks good to me. Please do merge this when ready.

**Pull request merged**